src/travel-details-screens/components/Trip.tsx

```diff
@@ -51,6 +72,16 @@ const Trip: React.FC<TripProps> = ({
                 )}
                 leg={leg}
                 testID={'legContainer' + index}
+                realtimePosition={tripVehiclePosition}
+                onExpand={() =>
+                  onPressDetailsMap({
+                    legs: tripPattern.legs,
+                    fromPlace: tripPattern.legs[0].fromPlace,
+                    toPlace:
+                      tripPattern.legs[tripPattern.legs.length - 1].toPlace,
+                    _vehiclePosition: tripVehiclePosition,
+                  })
+                }
```

Comment:
Do we actually need two parameters here? If it is just for showing a button, then I think we can encapsulate this into a single parameter with something like:
`onExpand={tripVehiclePosition ? () => ... : undefined}`

Also I think the variable name should be a little more precise than `onExpand`. Maybe something like `onPressShowInMap` or something.